drivers/dma/dma_stm32.c

```diff
@@ -28,15 +28,16 @@ static void dmamux_stm32_irq_handler(void *arg)
 	struct device *dev = arg;
 	struct dmamux_stm32_data *data = dev->driver_data;
 	const struct dmamux_stm32_config *config = dev->config->config_info;
-	DMAMUX_Channel_TypeDef *dmamux = (DMAMUX_Channel_TypeDef *)(config->base);
```

Comment:
Move these fixes directly in faulty commit.

Reply:
done